src/JsonSchema/SchemaFactory.php

```diff
@@ -88,6 +88,19 @@ public function buildSchema(string $className, string $format = 'json', string $
             return $schema;
         }
 
+        $isJsonMergePatch = 'json' === $format && 'PATCH' === $operation->getMethod() && Schema::TYPE_INPUT === $type;
+        $skipRequiredProperties = false;
+
+        if ($isJsonMergePatch) {
+            if (null === ($skipRequiredProperties = $operation->getExtraProperties()['patch_skip_schema_required_properties'] ?? null)) {
+                trigger_deprecation('api-platform/core', '3.4', "Set 'patch_skip_schema_required_properties' on extra properties as API Platform 4 will remove required properties from the JSON Schema on Patch request.");
+            }
+
+            if (true === $skipRequiredProperties) {
+                $definitionName .= self::PATCH_SCHEMA_POSTFIX;
```

Comment:
Undefined constant

Reply:
Also, I think if we add this postfix here on Patch operations, it should also be added on other operations (Post, Put, Delete, Get, GetCollection, etc.)